### DISCUSSION TITLE
Getting the weird error when deploying the nextjs14 application to the vercel (error message provided below)

### DISCUSSION THREAD

**Discussion by 0xsaad76** [vercel/community#7315] [Help] [answered]
### Summary

![image](https://github.com/vercel/community/assets/115649011/f31ec7af-6e60-4b73-8b0c-1182dc06fd9e)

im using nextj14, prisma and nextauth for the authentication

the codebase [link ](https://github.com/spider076/Cedium)

please help me solve this weird bug 

### Steps to Reproduce

the deployed application link : [https://cedium.vercel.app/](https://cedium.vercel.app/)

these are the website vercel logs : 

![image](https://github.com/vercel/community/assets/115649011/63e81966-9bab-4915-96ab-2da0c1063c57)

**Comment by pawlean** [comment]
Hi @spider076! 

I saw from your screenshot that your app is running now. Did you manage to solve this? If so, would love to see your solution (and also for the community 😄)

**Reply by 0xsaad76** [reply, marked as answer]
Yes! The app is running smoothly now. The issue was with the next-auth environment. I had to add the "NEXTAUTH" variable to the production environment, and following this document https://nextjs-faq.com/fetch-api-in-rsc helped me solve the bug.

**Reply by pawlean** [reply]
Great to know, thanks for coming back! 😄

[a comment by an automated account is not shown]
